### ISSUE TITLE
The mod Stack Combinator (1.2.11) caused a non-recoverable error. Factorio v1.1.42

### ISSUE THREAD

**Issue by jflemieux** [modo-lv/factorio-mod-stack-combinator#42]
Hi, I encountered this issue after trying to load a game with Factorio's lastest version (1.1.42):

```
The mod Stack Combinator (1.2.11) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event stack-combinator::on_load()
Detected modifications to the 'global' table:
CRC before: 1369323176
CRC after: 1291657281.
This indicates the mod is not save/load stable and not multiplayer safe.
on_load() should never change the global table.
```

I see that issue #41 is still opened, but for another version of the mod, sorry if it is the same issue.

I am available and would gladly help resolve the issue by providing more info/files or testing potential solutions.

(Thanks for this great mod, it's part of my "must have" list of mods)

**Comment by modo-lv** [comment]
Could you please share the save file? I don't know when I'll have time to look into this, but I will need a reproducible example.

**Comment by jflemieux** [comment]
[krastorio2.zip](https://github.com/modo-lv/factorio-mod-stack-combinator/files/7368749/krastorio2.zip)

Here is the save, as you can guess by the name, I was doing a Krastorio2 run.

**Important note:** I tested the save with only your mod active, with the same error.

Also: On my last play (while it was working), I remember that I used the settings copy hotkey (Shift+Left/Right click) on a stack combinator, everything seemed to work fine, but I preferred to mention it since it's the only "new" action I did before the corrupted save file.

So either that of the new Factorio version..

**Comment by ladenedge** [comment]
Any chance progress has been made on this issue?  I just received the same problem.  

![image](https://user-images.githubusercontent.com/2699775/148624018-1243bb2d-4fcf-406e-b92a-046711fff84f.png)

In case it matters, I just started receiving this window at load-game time as well:

![image](https://user-images.githubusercontent.com/2699775/148624093-377f930b-114d-4b5b-ad1c-724e5ac35bf9.png)

When I select "Sync Mods," Factorio reports all mods as "Correct" (including Stack Combinator).

Other info:

- I saved and exited about 1 hour before trying the load.  Nothing strange about it.
- All of my auto saves suffer the same problem.
- During my last playthrough, I pasted many new Stack Combinators, but I didn't do anything strange with them (that I can think of).

Save link: https://www.dropbox.com/s/3icnp5k6tggyh7z/Infinity.zip?dl=0
